### ISSUE TITLE
Revisit Seamless options for KNMI, DMI and MetNO model

### ISSUE THREAD

**Issue by guidocioni** [open-meteo/open-meteo#1094]
The "seamless" definition is transparently applied to all models that have different nested domains/configurations covering similar areas. However, for KNMI, DMI and MetNo there is an additional blending with ECMWF-IFS.

Although I understand the logic behind doing that, I think this is confusing, as the `seamless` name is used for completely different configurations: the user (and developer) may be lead to think that e.g. ICON seamless also contains IFS data, while it does not.
I think it should be better to have 

- KNMI **Seamless** (using Netherhlands + Europe)
- DMI **without Seamless** (only Europe domain)
- MetNO **without Seamless** (only Nordic domain)

In theory the best match for locations inside those domains should return a blending of the DMI/KNMI/MetNo model + IFS, so there's no need to have an additional seamless option.

At least, these are my two cents :)

**Comment by patrick-zippenfenig** [comment]
Absolutely! I've already been considering a new mechanism to better accommodate more models and provide transparency regarding which models are actually used. When using `best_match`, GFS is integrated as well to extend forecasts up to 16 days.

"Blending" is challenging. I’m not in favor of simply averaging multiple models. While this can improve results statistically, it often causes inconsistencies across forecast elements like temperature, humidity, cloud cover, radiation, or wind.

I’ve started collecting data, including "ECMWF IFS 4D long-window assimilation," and will soon integrate satellite observation datasets as reference data. This will allow for validation of all weather models and moving towards a generic blend of all deterministic and ensemble models.
